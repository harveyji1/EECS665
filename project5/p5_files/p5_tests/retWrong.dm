fn: () int{
    hello: bool;
    return hello;
}

main: () void{
    fn();
}