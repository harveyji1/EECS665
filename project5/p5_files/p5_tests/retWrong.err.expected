FATAL [3,12]-[3,17]: Bad return value
Type Analysis Failed